proto/p4/config/v1/p4types.proto

```diff
@@ -189,11 +201,11 @@ message P4EnumTypeSpec {
   message Member {
     string name = 1;
     repeated string annotations = 2;
```

Comment:
Why isn't this deleted? (same comment for other occurrences of `string annotations`

Reply:
Good catch, fixed :)